client/src/app/site/site.component.scss-theme.scss

```diff
@@ -29,43 +29,42 @@
         /** change the nav-toolbar to the darker nuance of the current theme*/
         .nav-toolbar {
             height: 80px;
-            background-color: mat-color($primary, darker);
+            background: white;
 
             mat-toolbar-row {
                 height: 80px;
             }
         }
 
+        .mat-drawer-inner-container {
+            scrollbar-width: none; /* hide scrollbars in Firefox */
+            -ms-overflow-style: -ms-autohiding-scrollbar; /* hide scrollbars in Edge until the scrollable area is hovered */
+        }
+
+        .mat-drawer-inner-container::-webkit-scrollbar {
+            display: none; /* hide scrollbars in webkit browsers */
+        }
+
+        .main-nav {
+            border-top-width: 1px;
+            border-top-style: solid;
+            border-top-color: rgba(0, 0, 0, 0.12);
```

Comment:
do not use hard colors in themes.
`border-top-color: mat-color($primary, hover);`